### PR TITLE
Replace status bar item application update with different tray icon based on status of the update

### LINKED ISSUE #5763: Tray Icon should show animated spinner while downloading update
**What would you like to be added**:
The tray icon currently is just the Lens logo or when an update is ready to be installed it is the Lens logo with an exclamation mark (`!`).

It would be great to have an animated spinner as part of the tray icon while an update is being downloaded.

**Why is this needed**:
Some better UX for auto update.

### PR DESCRIPTION
**Pull request by jansav**
Introduces placeholder tray icon for checking of updates. 

Review commit by commit. Makes it a lot easier.

### CODE REVIEW

File: build/generate-tray-icons.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ const size = Number(process.env.OUTPUT_SIZE || "16");
 const outputFolder = process.env.OUTPUT_DIR || "./build/tray";
 const inputFile = process.env.INPUT_SVG_PATH || "./src/renderer/components/icon/logo-lens.svg";
 const noticeFile = process.env.NOTICE_SVG_PATH || "./src/renderer/components/icon/notice.svg";
+const spinnerFile = process.env.SPINNER_SVG_PATH || "./src/renderer/components/icon/ssh.svg";
```
Comment (jansav) [inline build/generate-tray-icons.ts]: Notice that we are using placeholder icon here.

File: src/main/application-update/tray-icons/checking-for-updates-tray-icon.injectable.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
+/**
```
Comment (Nokel81) [inline src/main/application-update/tray-icons/checking-for-updates-tray-icon.injectable.ts:1]: Why are these here under and not under `tray/menu-icon` like the `normal-tray-icon`?

**Review by Nokel81 — APPROVED** [review]

**Review by jim-docker — APPROVED** [review]
